### PR TITLE
feat: notification sound setting ui 

### PR DESCRIPTION
**Pull request by hmquannnnn**
notification sound setting ui with mock data
task: https://github.com/nccasia/mezon-fe/issues/1797

### CODE REVIEW

File: libs/components/src/lib/components/ClanSettings/NotificationSoundSetting/index.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      ...sound,
+      isSelected: sound.id === selectedSound.id,
+    }));
+    setSoundList(updatedSoundList);
+    setCurrentSound(selectedSound);
+  };
+  
+  const handlePlaySound = (sound: SoundType) => {
+    setPlayingSound(sound);
+    console.log (playingSound)
```
Comment (anbuihoang) [inline libs/components/src/lib/components/ClanSettings/NotificationSoundSetting/index.tsx]: Remove log

File: libs/components/src/lib/components/ClanSettings/NotificationSoundSetting/index.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import SoundItem from "./SoundItem";
+// import * as  from "./../../../../../../assets/src/assets/videos"
```
Comment (anbuihoang) [inline libs/components/src/lib/components/ClanSettings/NotificationSoundSetting/index.tsx]: Remove cmt

Comment (hmquannnnn) [reply libs/components/src/lib/components/ClanSettings/NotificationSoundSetting/index.tsx]: done

File: libs/components/src/lib/components/ClanSettings/NotificationSoundSetting/index.tsx (hunk 1 above)
Comment (hmquannnnn) [reply libs/components/src/lib/components/ClanSettings/NotificationSoundSetting/index.tsx]: done

**Pull request merged**